.travis.yml

```diff
@@ -19,28 +19,37 @@ install:
   # - chmod +x ~/bin/terraform-docs
   # awscli
   - pip install awscli --upgrade --user
-  # gotest
-  - go get github.com/rakyll/gotest
-  - go install github.com/rakyll/gotest
+  - aws configure set aws_access_key_id     $CI1_AWS_ACCESS_KEY_ID     --profile cztack-ci-1
+  - aws configure set aws_secret_access_key $CI1_AWS_SECRET_ACCESS_KEY --profile cztack-ci-1
+  - aws --profile cztack-ci-1 sts get-caller-identity
+  - aws configure set aws_access_key_id     $CI2_AWS_ACCESS_KEY_ID     --profile cztack-ci-2
+  - aws configure set aws_secret_access_key $CI2_AWS_SECRET_ACCESS_KEY --profile cztack-ci-2
+  - aws --profile cztack-ci-2 sts get-caller-identity
+
+# - name: lint
+  # stage: check
+  # script: make lint
+# terraform-docs is incompatible with 0.12 for now
+# - stage: check
+#   script: make check-docs
+
+script:
+  - travis_wait 45 make test-ci
 
-jobs:
-  include:
-    - stage: check
-      script: make lint
-    # terraform-docs is incompatible with 0.12 for now
-    # - stage: check
-    #   script: make check-docs
-    - stage: check
-      before_script:
-        - aws configure set aws_access_key_id     $CI1_AWS_ACCESS_KEY_ID     --profile cztack-ci-1
-        - aws configure set aws_secret_access_key $CI1_AWS_SECRET_ACCESS_KEY --profile cztack-ci-1
-        - aws --profile cztack-ci-1 sts get-caller-identity
-        - aws configure set aws_access_key_id     $CI2_AWS_ACCESS_KEY_ID     --profile cztack-ci-2
-        - aws configure set aws_secret_access_key $CI2_AWS_SECRET_ACCESS_KEY --profile cztack-ci-2
-        - aws --profile cztack-ci-2 sts get-caller-identity
-      script: travis_wait 45 make test
 env:
+  jobs:
+    - TEST_BUCKET_INDEX=0
```

Comment:
I guess I don't quite understand how these interact with the `script` portion in travis syntax

Reply:
It is confusing and I am ready to try something besides travis :)